### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *hawkeye-sterling-v2* failed.

### ISSUE THREAD

**Issue by trex0092** [trex0092/compliance-analyzer#415]
### <span aria-hidden="true">❌</span> Deploy Preview for *hawkeye-sterling-v2* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 462153fa37ae94fba883ccf0ad21ae7d32429fb3 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/hawkeye-sterling-v2/deploys/69e67731b7e58c2687b8b224 |

_Originally posted by @netlify[bot] in https://github.com/trex0092/compliance-analyzer/pull/407#issuecomment-4282408564_
